### DISCUSSION TITLE
Docker container?

### DISCUSSION THREAD

**Discussion by TornaxO7** [ActivityWatch/activitywatch#1079] [Q&A] [answered]
Hello, may I ask if there's an official docker container to run activity watch?

**Comment by 0xbrayo** [comment, marked as answer]
No. There's no official docker container. I guess it would not be too hard to wrap the server in a docker container though.

**Comment by ErikBjare** [comment]
This has been asked and answered before: https://github.com/ActivityWatch/activitywatch/issues/166
